### PR TITLE
actually pin k8s.io/* to kubernetes-1.16.0

### PR DESCRIPTION
**Pull request by sanchezl**
Use `replace` to actually pin the `k8s.io` deps.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,18 +6,27 @@ require (
...
 	github.com/pkg/profile v1.3.0 // indirect
 	github.com/prometheus/client_golang v0.9.4
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
-	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783 // indirect
-	k8s.io/apimachinery v0.0.0-20190913080033-27d36303b655
-	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
-	k8s.io/component-base v0.0.0-20190918160511-547f6c5d7090
+	k8s.io/api v0.0.0-kubernetes-1.16.0
```
Comment (sttts) [inline go.mod]: do a `go mod tidy`. I believe these tags will be replaced again with SHAs. Go mod knows what is good for you :-/

Comment (sanchezl) [reply go.mod]: I purposely put that tag in there as a breadcrumb of sorts to document my intent. `go mod` leaves it alone since it is being `replace`d.

**Comment by sanchezl** [comment]
No longer needed.

**Pull request closed without merging**